### ISSUE TITLE
 Repeated scenes in Landsat order download.

### ISSUE THREAD

**Issue by robmartz** [CONABIO/antares3#65]
It has been observed that a downloaded order can have more than one scene repeated, the only difference is the processing date. For example:

```
LE070280442012052301T1-SC20180604135627
LE070280442012052301T1-SC20180606081922
LE070280442012052301T1-SC20180606144539
```
To avoid redundancy of scenes, it is necessary to implement a filter in `create_order` command. The dictionary that the query returns, must be edited before sending it as a request.

**Comment by palmoreck** [comment]
Also this happens in names of *.tar.gz

**Comment by palmoreck** [comment]
👆  which are the files that are downloaded in the file system

**Comment by palmoreck** [comment]
solved by 3b0034d8c6ce9991dc02039974f05c4242dda6c0
